packages/vuetify/src/components/VTextField/VTextField.ts

```diff
@@ -377,7 +377,7 @@ export default baseMixins.extend<options>().extend({
       return this.$createElement('input', {
         style: {},
         domProps: {
-          value: this.lazyValue,
+          value: Object.is(this.lazyValue, -0) ? '-0' : this.lazyValue,
```

Comment:
I can't find a better way to implement this, but we shouldn't perform this comparison if the `type` isn't `number`.

Reply:
I added a type check before the comparison but I am not sure this follows the code style of the project.

This will re-introduce the bug if the VTextField type is "text" and the v-model has the .number modifier (second field in the updated playground).  This might not be an issue because I can't think of a scenario where you would use the number modifier without the number type.

Reply:
You'd be surprised lol

Reply:
> You'd be surprised lol

I checked our own projects, and it is indeed filled with them.